src/header/launchbar/Launchbar.scss

```diff
@@ -91,3 +91,24 @@
   margin-left: 0.5rem;
   margin-right: 0.5rem;
 }
+
+.toolsIconWrapper {
+  position: relative;
+}
+
+.jobStatus {
+  border-radius: 10px;
+  height: 18px;
+  width: 18px;
+  position: absolute;
+  top: -9px;
+  left: 23px;
```

Comment:
Since you are putting the dot in the top right corner, the `right` property would be more appropriate. It won't depend on the width of the button, as it currently seems to be.

Reply:
✅